### PR TITLE
Update ProductExportFileHandler.php

### PR DESCRIPTION
**Pull request by sebastianlenz**
Regenerate always if cache is expired. Even if product export was generated by cronjob.

### 1. Why is this change necessary?
Without this change, the flag "-f" or "--force" of bin/console product-export:generate are ignored.

### 2. What does this change do, exactly?
It changes the way invalid files are detected.

### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [ ] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-Implement-New-Changelog.md) with all necessary information about my changes
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [ ] I have read the contribution requirements and fulfil them.

### CODE REVIEW

**Comment by shyim** [comment]
Could you add a test and a changelog file?

File: src/Core/Content/ProductExport/Service/ProductExportFileHandler.php [hunk 1]
```diff
@@ -59,8 +59,13 @@ public function isValidFile(string $filePath, ExportBehavior $behavior, ProductE
         if (!$this->fileSystem->has($filePath)) {
             return false;
         }
+        
+        // Regenerate always if cache is expired. Even if product export was generated by cronjob.
+        if($this->isCacheExpired($behavior, $productExport)) {
```
Comment (shyim) [inline src/Core/Content/ProductExport/Service/ProductExportFileHandler.php:64]: Could you add a test and a changelog file?

**Review by shyim — CHANGES_REQUESTED** [review]

File: src/Core/Content/ProductExport/Service/ProductExportFileHandler.php (hunk 1 above)
Comment (sebastianlenz) [reply src/Core/Content/ProductExport/Service/ProductExportFileHandler.php:64]: @shyim i can implement a changelog. I'm not sure how to write a test for this.

Comment (shyim) [reply src/Core/Content/ProductExport/Service/ProductExportFileHandler.php:64]: Hey sorry for the late response. I was in vacation.

Create a new class ProductExportFileHandlerTest and use the IntegreationTestBehaviour.

With `$this->getContainer()->get(ProductExporter::class)` you get the exporter service and you can call then in the test methods the public method and assert what you should

**Comment by shyim** [comment]
Hey,

I will close this as it is inactive. Feel free to create a new PR when you have added the incomplete parts

**Pull request closed without merging**